src/components/NewClusterModal.js

```diff
@@ -442,16 +465,40 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
                             value: customEnvImage,
                             onChange: customEnvImage => this.setState({ customEnvImage })
                           },
-                          error: customEnvImage && isCustomImageInvalid && 'Not a valid image'
+                          error: customEnvImage && this.isInputtedImageInvalid(customEnvImage) && 'Not a valid image'
                         })
                       ])
                     ])
                   ]),
                   div({ style: { margin: '0.5rem' } }, [
-                    h(Link, { href: imageInstructions, ...Utils.newTabLinkProps }, ['Custom notebook environments']),
+                    'Custom environments',
                     span({ style: { fontWeight: 'bold' } }, [' must ']),
                     ' be based off one of the ',
-                    h(Link, { href: terraBaseImages, ...Utils.newTabLinkProps }, ['Terra base images.'])
+                    h(Link, { href: terraBaseImages, ...Utils.newTabLinkProps }, ['Terra Jupyter Notebook base images']),
+                    ' or a ',
+                    h(Link, { href: zendeskImagePage, ...Utils.newTabLinkProps }, ['Project-Specific image'])
+                  ])
+                ])
+              }],
+              [PROJECT_SPECIFIC_MODE, () => {
+                return h(Fragment, [
+                  div({ style: { lineHeight: '1.5' } }, [
+                    'Some consortium projects, such as ',
+                    h(Link, { href: rstudioBaseImages, ...Utils.newTabLinkProps }, ['AnVIL']),
+                    ', have created environments that are specific to their project. If you want to use one of these: ',
+                    div({ style: { marginTop: '0.5rem' } }, ['1. Find the environment image (',
+                      h(Link, { href: zendeskImagePage, ...Utils.newTabLinkProps }, ['view image list']),
+                      ') ']),
```

Comment:
nit: this code spacing is confusing when coupled with this particular text. I think expanding the children array fully would help, e.g.:
```
div([
  X,
  X,
  X
])
```